mvpa2/misc/neighborhood.py

```diff
@@ -253,6 +256,7 @@ def __init__(self, radius, inner_radius, **kwargs):
                   "than the radius (got %g)" % (inner_radius, radius)
         Sphere.__init__(self, radius, **kwargs)
         self._inner_radius = inner_radius
+        self.include_center = include_center
```

Comment:
now also adjust ```__repr__``` below to add ```include_center ``` when it is not default... check other repr's for examples